experiments/compositions/samples/Eventarc/eventarc-composition.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: composition.google.com/v1alpha1
+kind: Context
+metadata:
+  name: context
+  namespace: config-control
+spec:
+  project: # Replace with your project ID
```

Comment:
I would remove it from here.
I would add it to the readme instructions instead.
Or part of Facade.